pact/fixed-quote-royalty-policy.pact

```diff
@@ -154,7 +161,10 @@
           , 'recipient-guard := recipient-guard:guard
           }
           (let ((sale-price (* amount price)))
-            (fungible::transfer-create buyer creator creator-guard (* sale-price royalty-rate))
+            (if
+              (> (* sale-price royalty-rate) 0.0)
+              (fungible::transfer-create buyer creator creator-guard (* sale-price royalty-rate))
+              true)
```

Comment:
```suggestion
              "No royalty")
```

Reply:
We should `typecheck` this module (and the other policies) in the repl and fix any type errors